include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -97,6 +97,7 @@ class CloudEnvOptions {
         keep_local_log_files(_keep_local_log_files),
         manifest_durable_periodicity_millis(
             _manifest_durable_periodicity_millis),
+        purger_periodicity_millis(_purger_periodicity_millis),
```

Comment:
copy `_purger_periodicity_millis` argument to `purger_periodicity_millis` field.